src/fidesops/ops/models/privacy_request.py

```diff
@@ -730,6 +732,18 @@ def hash_value(
         return hashed_value
 
 
+class Consent(Base):
+    """The DB ORM model for Consent."""
+
+    provided_identity_id = Column(String, ForeignKey(ProvidedIdentity.id), unique=True)
+    regulation = Column(EnumColumn(Regulations))
```

Comment:
This doesn't make too much sense to me for a couple reasons- one is that consent is a distinct consult from regulation (you don't collect consent for GDPR, you collect for a specific use of a customer's data), and second is that we can't realistically maintain an enum of regulations (this regulations enum is already way too limited)

We should remove the regulation field entirely from this so it didn't confuse 👍